LongWarOfTheChosen/Src/LW_FactionBalance/Classes/X2Effect_PaleHorse_LW.uc

```diff
@@ -0,0 +1,29 @@
+//---------------------------------------------------------------------------------------
+//  FILE:    X2Effect_PaleHorse_LW
+//  AUTHOR:  Peter Ledbrook
+//  PURPOSE: Modified version of Pale Horse (Soul Harvest) that can apply to all
+//           weapons and has a base crit bonus in addition to the per-kill bonus.
```

Comment:
Not sure if this is the way to fix soul harvest, but it's hard to think of an interesting ramp up ability that isn't broken. maybe something like XCOM blood thirst, where the crit bonus is a lot bigger but only temporary?

Reply:
Maybe, but I think Soul Harvest is fine as it is now, especially as it applies to knives. I have to admit apathy on changing it further at this point. A temporary effect could be more interesting, but maybe try that at a later date.